### PR TITLE
WRP-6110: Add command for extra repos in ReadMe.md and update it in DocParser.js and prepareRaw.js

### PR DESCRIPTION
**Pull request by andreitirlalge**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Add command for extra repos in README.md and update DocParser.js and prepareRaw.js

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
README.md now has a command for extra repos and DocParser.js and prepareRaw.js are now updated.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-6110

### Comments
Enact-DCO-1.0-Signed-off-by: Andrei Tirla andrei.tirla@lgepartner.com

### CODE REVIEW

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -14,6 +14,12 @@ documentation from the Enact source:
 npm run parse
 ```
 
+Additional repos can be pulled into the docs using the following command line arg:
+
+```
+extra-repos   (e.g. npm run parse --extra-repos enactjs/agate#develop,enactjs/moonstone#3.2.5)
+```
+
```
Comment (daniel-stoian-lgp) [inline README.md:22]:
```suggestion
Additional repos can be pulled into the docs using the `extra-repos` command line arg:

```
e.g. npm run parse --extra-repos enactjs/agate#develop,enactjs/moonstone#3.2.5
```

```

File: README.md [hunk 2]
```diff
@@ -14,6 +14,12 @@ documentation from the Enact source:
 npm run parse
 ```
 
+Additional repos can be pulled into the docs using the `extra-repos` command line argument:
+
+```
+e.g. npm run parse --extra-repos enactjs/agate#develop,enactjs/moonstone#3.2.5
```
Comment (sjro) [inline README.md:20]:
```suggestion
e.g. npm run parse -- --extra-repos enactjs/agate#develop,enactjs/moonstone#3.2.5
```

File: scripts/DocParser.js [hunk 3]
```diff
@@ -7,7 +7,7 @@
 // * --no-static - Do not copy static documentation files
 // * --pattern <pattern> - Only search for modules that match the pattern specified
 //    NOTE: <pattern> looks like: \*moonstone\*, Button.js, or T\*.js
-// * --extra-repos <repo-list>   (e.g. --extra-repos enact/agate#develop,enact/moonstone#3.2.5)
+// * --extra-repos <repo-list>   (e.g. --extra-repos enactjs/agate#develop,enact/moonstone#3.2.5)
```
Comment (sjro) [inline scripts/DocParser.js:10]:
```suggestion
// * --extra-repos <repo-list>   (e.g. --extra-repos enactjs/agate#develop,enactjs/moonstone#3.2.5)
```

File: scripts/prepareRaw.js [hunk 4]
```diff
@@ -8,7 +8,7 @@
  * * `eslint-config-branch`
  *
  * Additional repos can be pulled into the docs using the following command line arg:
- * * `extra-repos`   (e.g. --extra-repos enact/agate#develop,enact/moonstone#3.2.5)
+ * * `extra-repos`   (e.g. --extra-repos enactjs/agate#develop,enact/moonstone#3.2.5)
```
Comment (sjro) [inline scripts/prepareRaw.js:11]:
```suggestion
 * * `extra-repos`   (e.g. --extra-repos enactjs/agate#develop,enactjs/moonstone#3.2.5)
```

**Pull request merged**